app/routes/api.data.votes.ts

```diff
@@ -0,0 +1,103 @@
+import { Prisma } from "@prisma/client"
```

Comment:
This endpoint is not actually used anywhere rn, started with looking at political affiliation on a vote basis but moved to edge basis.

Leaving it in case it'll be useful later